src/fairchem/core/graph/radius_graph_pbc.py

```diff
@@ -326,14 +312,41 @@ def radius_graph_pbc(
     return edge_index, unit_cell, num_neighbors_image
 
 
+def canonical_pbc(data, pbc: torch.Tensor | None):
```

Comment:
Do we still need all this if we are always running `radius_graph_fn` over a single system at a time?

Reply:
V2 is supposed to be fine with batched data. I dont know if we should take out the batching logic for pbc graph gen in general. V2 specifically addressed batching so it would be a feature of V2 we would not be using. I am down for discussing it, but feels like maybe should be a separate PR/Task? 

I added this so at least both v1 and v2 of pbc graph gen share the same pbc logic, instead of copy paste which diverged in our current implementation.